plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -147,6 +147,13 @@ public void testCreateTableWithCommaPropertyColumn()
         }
     }
 
+    @Test
+    public void testCreateTableWithNonExistingPrimaryKey()
+    {
+        assertQueryFails("CREATE TABLE test_create_table_non_existing_primary_key (a bigint, b double) with (primary_key = ARRAY['c'])",
+                "Column 'c' specified in property 'primary_key' doesn't exist in table");
+    }
```

Comment:
Let's add below assertions to this test method:
* Specify existing column with uppercase in primary_key. e.g. A
* Specify `dummy_id` column explicitly in primary_key

Reply:
> @sahoss Please feel free to ping me after addressing comments.

@ebyhr done.
deleted the unused branch in https://github.com/trinodb/trino/pull/16266/commits/328e5a650d773478fb17b99f608dd243804ae374

Reply:
Reminder.

Reply:
@ebyhr this is done. please take a look to see if this was how you meant it?

```
assertQueryFails("CREATE TABLE test_create_table_non_existing_primary_key (a bigint, b bigint, c bigint) with (primary_key = ARRAY['A', 'b', 'dummy_id'])",
                "Column 'dummy_id' specified in property 'primary_key' doesn't exist in table")
```

Reply:
Please don't mix up the assertions.

Reply:
@ebyhr for `Specify existing column with uppercase in primary_key. e.g. A`
i.e. 
```
CREATE TABLE test_create_table_non_existing_primary_key (a bigint, b bigint, c bigint) with (primary_key = ARRAY['A'])
```
is the expected behavior to fail the request? I see that we are lowercasing the column names(in the table definition), do we also lowercase the attributes in primary key property or do we fail the request?

Reply:
@ebyhr 
`Specify dummy_id column explicitly in primary_key` 
is this a special case of sorts or does the current test case exercise this scenario? ptal at the previous question as well.

Reply:
I expect both scenarios fail.

Reply:
@ebyhr please take a look. have added 2 separate failure assertions.

Reply:
`dummy_id` is a special naming in the connector. I would recommend splitting the 1st `assertQueryFails` into two assertions.

Reply:
> `dummy_id` is a special naming in the connector. I would recommend splitting the 1st `assertQueryFails` into two assertions.

@ebyhr done. please take a look.

Reply:
Could you update like below? 
```java
    @Test
    public void testCreateTableWithNonExistingPrimaryKey()
    {
        String tableName = "test_invalid_primary_key" + randomNameSuffix();
        assertQueryFails("CREATE TABLE " + tableName + "(a bigint) WITH (primary_key = ARRAY['not_existing_column'])",
                "Column 'not_existing_column' specified in property 'primary_key' doesn't exist in table");

        assertQueryFails("CREATE TABLE " + tableName + "(a bigint) WITH (primary_key = ARRAY['dummy_id'])",
                "Column 'dummy_id' specified in property 'primary_key' doesn't exist in table");

        assertQueryFails("CREATE TABLE " + tableName + "(a bigint) WITH (primary_key = ARRAY['A'])",
                "Column 'A' specified in property 'primary_key' doesn't exist in table");
    }
```